profiles/common/modules/custom/tmgmt_poetry/plugins/tmgmt_poetry.ui.inc

```diff
@@ -312,9 +312,11 @@ class TMGMTPoetryTranslatorUIController extends TMGMTDefaultTranslatorUIControll
     $form['delai'] = array(
       '#type' => 'date_popup',
       '#title' => t('Requested delivery date'),
-      '#default_value' => (!empty($job->settings['delai']) ? $job->settings['delai'] : date('Y-m-d', strtotime('+1 month'))),
+      '#default_value' => '',
       '#date_format' => 'd/m/Y',
-      '#date_year_range' => '-3:+3',
+      '#date_year_range' => '-0:+1',
+      '#required' => TRUE,
+      '#weight' => -10,
```

Comment:
The date field is the only mandatory field that you need to fill. To improve UX after internal discussion we decided to have this field as a first in the field group.